flavors/vulnerability.go

```diff
@@ -96,16 +86,17 @@ func (bt *vulnerability) Run(b *beat.Beat) error {
 
 	// TODO: Should trigger first cycle without waiting for ticker.
 	bt.log.Warn("vulnerability.Run ticker running for period ", bt.config.Period)
-	workerTicker := time.NewTicker(bt.config.Period)
+	workerTicker := time.NewTicker(1)
```

Comment:
Instead of resetting the ticker, what do u think about wrapping the logic we have under `workerTicker.C` in a function and just calling it before the for loop?